libsolidity/analysis/DocStringTagParser.cpp

```diff
@@ -47,6 +47,65 @@ bool DocStringTagParser::parseDocStrings(SourceUnit const& _sourceUnit)
 	return errorWatcher.ok();
 }
 
+bool DocStringTagParser::validateDocStringsUsingTypes(SourceUnit const& _sourceUnit)
+{
+	auto errorWatcher = m_errorReporter.errorWatcher();
+
+	SimpleASTVisitor visitReturns([](ASTNode const&) { return true; }, [&](ASTNode const& _node)
+	{
```

Comment:
I'd wrap it this way. This clearly shows that we have two lambdas passed in parameter. The current line looked to me as a weird function declared in the middle of another function at first. 
```suggestion
	SimpleASTVisitor visitReturns(
		[](ASTNode const&) { return true; },
		[&](ASTNode const& _node)
		{
```